games/hanabi/src/api/index.ts

```diff
@@ -17,6 +19,10 @@ export type Colour = typeof colours[number];
 export const numbers = [1, 2, 3, 4, 5] as const;
 export type Number = typeof numbers[number];
 export const maxCardNum = numbers[numbers.length - 1];
+export const getColours = (gameType: GameType): Colour[] =>
+  gameType === "rainbow"
+    ? colours.slice()
```

Comment:
Why are you cloning?

Reply:
oh, this was a weird hack to get around a type error that I then forgot about - will look into a better fix (sorry, clicked merge a few minutes ago, so will address any comments in a new pr)